src/metabase/upload/parsing.clj

```diff
@@ -113,11 +117,14 @@
          (str/trim)
          (remove-currency-signs)
          (parse-plain-number number-separators))
-    (catch Throwable e
-      (throw (ex-info
-              (tru "{0} is not a recognizable number" s)
-              {}
-              e)))))
+    (catch Exception _
+      (throw (IllegalArgumentException. (tru "{0} is not a recognizable number" s))))))
+
+(defn parse-as-int
```

Comment:
it seems weird to me to convert the value to str in this function. 

we either should change the same for it to makes more sense, or we should convert it to str when we use it.

Reply:
Fixed with bc5b7d44cb08d8d82aeed81a967d82ecdba6e22c